CLAUDE.md

```
@@ -168,4 +168,45 @@ yarn update-protobuf  # Updates protobuf definitions in hd-transport
 ```
 
 ### Docker Development Environment
-The project includes Docker support for consistent development environments across different platforms.
\ No newline at end of file
+The project includes Docker support for consistent development environments across different platforms.
+
+## Expo-Example Routing Configuration
+
+### Overview
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)